backend/src/email/routes/email-campaign.routes.ts

```diff
@@ -88,6 +88,16 @@ const duplicateCampaignValidator = {
   }),
 }
 
+const tesseractCampaignValidator = {
+  [Segments.BODY]: Joi.object({
+    url: Joi.string()
+      .pattern(/vault.gov.sg/)
+      .uri({ scheme: 'https' })
+      .trim()
+      .required(),
+  }),
```

Comment:
Not sure if this is a well defined validator? I also check for valid vault domain in the `isValidVaultUrl` middleware